### ISSUE TITLE
SharedPtr-style template types not having their `<>` removed unless they have a namespace

### ISSUE THREAD

**Issue by nornagon** [bytedeco/javacpp#181]
[javacpp-ptr-namespace-repro.tar.gz](https://github.com/bytedeco/javacpp/files/997222/javacpp-ptr-namespace-repro.tar.gz)

This C++:

```c++
template <typename T> class sp { };
struct Foo {};
struct Bar { sp<Foo> foo; };
```

With this InfoMap:
```java
        infoMap.put(new Info("sp").skip().annotations("@Ptr"));
```

Produces this Java:
```java
public static class Bar extends Pointer {
    /* ... */
  public native @Ptr @ByRef <Foo> foo(); public native Bar foo(<Foo> foo);
}
```

But this C++:
```c++
namespace foo { template <typename T> class sp { }; }
struct Foo {};
struct Bar { foo::sp<Foo> foo; };
```
with this InfoMap:
```java
        infoMap.put(new Info("foo::sp").skip().annotations("@Ptr"));
```
produces this Java:
```java
public static class Bar extends Pointer {
    /* ... */
 public native @Ptr Foo foo();
}

**Comment by saudet** [comment]
This should be fixed with the commit above. Thanks for reporting!

**Comment by saudet** [comment]
Fix released in version 1.3.3. Enjoy!
